main.py

```diff
@@ -0,0 +1,14 @@
+def define_env(env):
+    """
+    This is the hook for the variables, macros and filters.
+    """
+
+    @env.macro
+    def full_bc(lmdb_size_full, no):
+        "Recommended Storage"
+        return lmdb_size_full * no
+
+    @env.macro
+    def pruned_bc(lmdb_size_pruned, no):
+        "Recommended Storage"
+        return lmdb_size_pruned * no
```

Comment:
Semantics: so we have 2 functions that take 2 arguments, and return arg1 * arg1 (2 functions doing the same thing). We should just have one function called "multiply" with inputs arg1 and arg2